main.tf

```diff
@@ -26,6 +26,7 @@ resource "aws_iam_user" "iam_user" {
 
 # ref https://www.terraform.io/docs/providers/aws/r/iam_access_key.html
 resource "aws_iam_access_key" "iam_user" {
+  count   = var.create_access_key == true ? 1 : 0
```

Comment:
think can just `count = var.create_access_key ? 1 : 0` ?

Reply:
fixed in https://github.com/GovTechSG/terraform-iam-service-user-gcc/pull/1/commits/2b21e247d6a4a956a80d1525c279f230f0c8d3a9